src/helpers/tests/geoChart/data.spec.ts

```diff
@@ -0,0 +1,681 @@
+// (C) 2020 GoodData Corporation
+import { Execution, VisualizationObject } from "@gooddata/typings";
+import { COLOR_ITEM, LOCATION_ITEM, SEGMENT_BY_ITEM, SIZE_ITEM, TOOLTIP_TEXT_ITEM } from "./fixtures";
+import { getExecutionResponse, getExecutionResult } from "../../../../stories/data/geoChart";
+import { getGeoData } from "../../geoChart/data";
+
+describe("geoChartData", () => {
+    it("should return geoData with empty bucket", () => {
+        const buckets: VisualizationObject.IBucket[] = [];
+        const execution: Execution.IExecutionResponses = {
+            executionResponse: getExecutionResponse(),
+            executionResult: getExecutionResult(),
+        };
+
+        const geoData = getGeoData(buckets, execution);
+        expect(geoData).toEqual({});
+    });
+
+    it("should return geoData with full bucket", () => {
+        const buckets: VisualizationObject.IBucket[] = [
+            LOCATION_ITEM,
+            SEGMENT_BY_ITEM,
+            TOOLTIP_TEXT_ITEM,
+            SIZE_ITEM,
+            COLOR_ITEM,
+        ];
+        const execution: Execution.IExecutionResponses = {
+            executionResponse: getExecutionResponse(true, true, true, true, true),
+            executionResult: getExecutionResult(true, true, true, true, true),
+        };
+
+        const geoData = getGeoData(buckets, execution);
+        expect(geoData).toEqual({
+            color: {
+                data: [
+                    528,
+                    6832,
+                    3294,
+                    8340,
+                    957,
+                    116,
+                    253,
+                    1047,
+                    7520,
+                    22220,
+                    18,
+                    2282,
+                    5602,
+                    150,
+                    1242,
+                    1782,
+                    335,
+                    2299,
+                    596,
+                    12064,
+                    869,
+                    433,
+                    1605,
+                    1121,
+                    570,
+                    11107,
+                    45703,
+                    8732,
+                    2627,
+                    1381,
+                    11564,
+                    2288,
+                    3500,
+                    3310,
+                    5302,
+                    3836,
+                    1938,
+                    1560,
+                    1766,
+                    3949,
+                    772,
+                    709,
+                    3060,
+                    838,
+                    2844,
+                    1719,
+                    4726,
+                    179,
+                    943,
+                    1005,
```

Comment:
could you please limit the test data to 10 items only?

Reply:
I would cause to update all related storybook images, we should consider to implement it in other separate ticket.

Reply:
we can do it later